contrib/redis-mixin/alerts/redis.libsonnet

```diff
@@ -30,7 +30,7 @@
           },
           {
             alert: 'RedisTooManyConnections',
-            expr: 'redis_connected_clients > 100',
+            expr: 'redis_connected_clients > %(redisConnectionsThreshold)' % $._config,
```

Comment:
```suggestion
            expr: 'redis_connected_clients > %(redisConnectionsThreshold)s' % $._config,
```

Reply:
jsonnet uses the same mini-language as python for string formatting, so you need to specify it's a string

https://docs.python.org/2/library/stdtypes.html#string-formatting